working/augmentation-libraries/feature-specification.md

```diff
@@ -459,14 +459,18 @@ augment int slowCalculation(int a, int b) {
 The augmentation replaces the augmented function’s body with the augmenting
 function’s body.
 Inside the augmenting function’s body, a special `augmented(…)` expression may
-be used to execute the original function body. That expression takes an
-argument list matching the original function's parameter list and returns the
+be used to execute the augmented function body. That expression takes an
+argument list matching the augmented function's parameter list and returns the
 function's return type.
 
 The augmenting function does not have to pass the same arguments to
 `augmented(…)` as were passed to it. It may invoke `augmented` once, more than
 once, or not at all.
 
+Augmenting function declarations may also omit the body, in order to only
+augment the metadata or doc comments of the function. In this case the body of
+the augmented member is not altered.
```

Comment:
We'll probably want to be clearer about what it means to "omit" the body of a function. Maybe:

> An augmenting function declaration may have an empty body (`;`) in order to only augment the metadata...